model/websocket_message.go

```diff
@@ -75,6 +75,11 @@ const (
 	WebsocketEventThreadFollowChanged                 = "thread_follow_changed"
 	WebsocketEventThreadReadChanged                   = "thread_read_changed"
 	WebsocketFirstAdminVisitMarketplaceStatusReceived = "first_admin_visit_marketplace_status_received"
+	WebsocketEventSubscribe                           = "subscribe"
+	WebsocketEventUnsubscribe                         = "unsubscribe"
+
+	WebsocketSubjectActivityFeed WebsocketSubjectID = "activity_feed"
+	WebsocketSubjectActiveCalls  WebsocketSubjectID = "active_calls"
```

Comment:
Let's add a comment here to add any new subject to the `IsValid` method as well to keep both in sync.

Reply:
I removed these to support dynamic subject ids.